src/main/java/bisq/price/spot/providers/CryptoYa.java

```diff
@@ -79,4 +71,25 @@ private CryptoYaMarketData fetchArsBlueMarketData() {
                 .bodyToMono(CryptoYaMarketData.class)
                 .block(Duration.of(30, ChronoUnit.SECONDS));
     }
+
+    public Set<ExchangeRate> last24HoursToExchangesRates(CryptoYaMarketData cryptoYaMarketData) {
+        long yesterdayTimestamp = Instant.now()
+                .minus(1, ChronoUnit.DAYS)
+                .getEpochSecond();
+
+        Stream<Pair<String, CryptoYaTicker>> exchangeNameWithTickerStream =
+                cryptoYaMarketData.exchangeNameWithTickerStream()
+                        .filter(exchangeNameWithTicker -> Objects.nonNull(exchangeNameWithTicker.getRight()));
+
+        return exchangeNameWithTickerStream
+                .filter(exchangeNameWithTicker -> exchangeNameWithTicker.getRight()
+                        .getTime() > yesterdayTimestamp)
+                .filter(exchangeNameWithTicker -> exchangeNameWithTicker.getRight().getAsk() > 0)
+                .map(exchangeNameWithTicker -> new ExchangeRate(
+                        "ARS",
+                        exchangeNameWithTicker.getRight().getAsk(),
+                        System.currentTimeMillis(),
+                        "CryptoYa: " + exchangeNameWithTicker.getLeft())
```

Comment:
Please could you uppercase `CryptoYa` to `CRYPTOYA`, or make a string constant for the provider name at the top of the class as it is used more than once.  The provider name is compared in ExchangeRateService when writing the exchange timestamp to metadata.

Reply:
Could you also change ExchangeRateService to use `startsWith` when [comparing the provider name](https://github.com/bisq-network/bisq-pricenode/blob/94b26749afd1d6f70823f11ea6118dd03c4bfb2f/src/main/java/bisq/price/spot/ExchangeRateService.java#L234-L236):


----

This change works (tested :+1: ) with the newly appended CRYPTOYA provider names:

     .filter(e -> e.getProvider().startsWith(provider.getName()))


----

Resultant metadata:


    "cryptoyaTs" : 1694809554486,
    "cryptoyaCount" : 19,